### ISSUE TITLE
False positive in no-conflicting-classes for tailwind-scrollbar utilities (scrollbar-thin + scrollbar-thumb-*/scrollbar-track-*)

### ISSUE THREAD

**Issue by jrparish** [sergioazoc/oxlint-tailwindcss#93]
## Summary

`no-conflicting-classes` reports a conflict on `scrollbar-color` when `scrollbar-thin` is combined with `scrollbar-thumb-*` / `scrollbar-track-*` (from the `tailwind-scrollbar` plugin). These utilities are designed to compose, so this is a false positive.

```
warning tailwindcss(no-conflicting-classes): "scrollbar-thin" and "scrollbar-thumb-core-gray-700" both affect "scrollbar-color". Only one applies — which wins depends on the generated stylesheet order, not the attribute order. Remove one.
```

## Environment

- `oxlint-tailwindcss`: 1.3.5
- `oxlint`: 1.74.0
- Tailwind CSS: v4 (CSS-first, entry point via `settings.tailwindcss.entryPoint`)
- `tailwind-scrollbar`: 4.0.2, registered through the JS config that the CSS entry pulls in with `@config '../tailwind.config.js'`

## Why it's a false positive

The two class families set *different* things:

- `.scrollbar-thin` declares the `scrollbar-color` **property**, consuming two custom properties:

  ```css
  .scrollbar-thin {
    scrollbar-width: thin;
    scrollbar-color: var(--scrollbar-thumb, initial) var(--scrollbar-track, initial);
  }
  ```

- `.scrollbar-thumb-*` / `.scrollbar-track-*` set the **custom properties**, not the `scrollbar-color` property (via `matchUtilities`):

  ```css
  .scrollbar-thumb-gray-700 { --scrollbar-thumb: #374151; }
  .scrollbar-track-gray-950 { --scrollbar-track: #030712; }
  ```

So `scrollbar-thin scrollbar-thumb-gray-700 scrollbar-track-gray-950` is the plugin's intended usage — `scrollbar-thin` reads the vars the other two supply. There is no `scrollbar-color` conflict; only `scrollbar-thin` writes that property. The rule appears to attribute `scrollbar-color` to the color utilities (either via a built-in taxonomy heuristic, or by mapping their `--scrollbar-thumb` / `--scrollbar-track` custom-property declarations onto the `scrollbar-color` shorthand).

## Reproduction

Tailwind v4 project with `tailwind-scrollbar` registered (here via `@config`-loaded JS config):

```jsx
<div className="scrollbar-thin scrollbar-thumb-gray-700 scrollbar-track-gray-950" />
```

`.oxlintrc.json`:

```jsonc
{
  "jsPlugins": ["oxlint-tailwindcss"],
  "settings": { "tailwindcss": { "entryPoint": "styles/globals.css" } },
  "rules": { "tailwindcss/no-conflicting-classes": "error" }
}
```

## Expected

No conflict reported — the classes compose. A utility that only writes a CSS custom property (e.g. `--scrollbar-thumb`) should not be treated as writing the property that consumes it (`scrollbar-color`).

## Actual

`no-conflicting-classes` flags `scrollbar-thin` against each `scrollbar-thumb-*` / `scrollbar-track-*` sibling.

**Comment by alitas** [comment]
Same thing is happening with `outline-[number]` and `outline-solid`
